app/client/components/cancel/cancellationReason.ts

```diff
@@ -0,0 +1,13 @@
+export interface CancellationReason {
+  reasonId: string;
```

Comment:
I know this isn't new, but we could introduce a type for these?

Reply:
done